extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/internal/discovery/TradfriDiscoveryParticipant.java

```diff
@@ -51,7 +51,7 @@ public ThingUID getThingUID(ServiceInfo service) {
         if (service != null) {
             String name = service.getName();
             if ((service.getType() != null) && service.getType().equals(getServiceType())
-                    && (name.matches("gw:([a-f0-9]{2}[-]?){6}"))) {
+                    && (name.matches("gw[:-]{1}([a-f0-9]{2}[-]?){6}.*"))) {
```

Comment:
Could you add a code comment that briefly explains the possible names of the gateway?